# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/sessionstore/test/browser_394759_perwindowpb.js
# Commit: d29c1fad7862
# Full Hash: d29c1fad7862d61ef7c66d5989caa5ed0af4f011
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2023-03-03 21:37:37
# Regressor Bug: 1423125
# File Overlap Count: 1
# Description:
#   Bug 1423125 - Restore previous session if undoCloseTab is called with no more tabs to restore. r=dao
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171377
# ==============================================================================

diff -r c81bb857ccff -r d29c1fad7862 browser/components/sessionstore/test/browser_394759_perwindowpb.js
--- a/browser/components/sessionstore/test/browser_394759_perwindowpb.js	Fri Mar 03 14:06:07 2023 +0000
+++ b/browser/components/sessionstore/test/browser_394759_perwindowpb.js	Fri Mar 03 14:06:43 2023 +0000
@@ -46,9 +46,7 @@
 
 add_setup(async function() {
   forgetClosedWindows();
-  while (ss.getClosedTabCount(window) > 0) {
-    ss.forgetClosedTab(window, 0);
-  }
+  forgetClosedTabs(window);
 });
 
 add_task(async function main() {